cmd/proxy/main.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 var (
-	configFile = flag.String("config_file", filepath.Join("..", "..", "config.toml"), "The path to the config file")
+	configFile = flag.String("config_file", filepath.Join("..", "..", "config.example.toml"), "The path to the config file")
```

Comment:
dont' you need to change dockerfiles so it does not do `copy config.example.toml config.toml` ?

Reply:
@michalpristas the Dockerfile overrides this default by passing the flag explicitly.